debian/homestead.init.d

```diff
@@ -190,6 +190,7 @@ get_daemon_args()
                      --server-name=\"$server_name\"
                      --impu-cache-ttl=$impu_cache_ttl
                      --hss-reregistration-time=$hss_reregistration_time
+                     --reg-max-expires=$reg_max_expires
```

Comment:
You should probably also get rid of the code at https://github.com/Metaswitch/homestead/blob/52ec9afcf653fbc7b890a954b9858ed23af27976/debian/homestead.init.d#L108 which protected against hss_reregistration_time being below reg_max_expires - I don't think it's helpful now. (Did you test live?)